nixos/modules/services/hardware/auto-epp.nix

```diff
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, ... }:
+with lib;
+
+let
+
+ cfg = config.services.auto-epp;
+ package = pkgs.auto-epp;
```

Comment:
Consider making this an [`mkPackageOption`](https://nixos.org/manual/nixpkgs/stable/#function-library-lib.options.mkPackageOption)